Wrapping/Python/PythonDocstrings.i

```diff
@@ -5361,6 +5366,163 @@ Destructor
 ";
 
 
+%feature("docstring") itk::simple::CoherenceEnhancingDiffusionImageFilter "
+
+Coherence enhanging diffusion and edge enhancing diffusion.
```

Comment:
Same comment as in java:
Coherence **enhancing** diffusion

Reply:
InsightSoftwareConsortium/ITKAnisotropicDiffusionLBR#19